### PR TITLE
SR1.1.1: Add support for showing standard deviation

### PR DESCRIPTION
**Pull request by mjjq**
Extend the graphs to show the standard deviation as required by SR1.1.1

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (0liverCrosby) [inline inflammation/models.py:58]: Function name could be a little more informative

File: tests/test_models.py [hunk 2]
```diff
@@ -37,3 +37,14 @@ def test_load_from_json(tmpdir):
...
     npt.assert_array_equal(result, [[1, 2, 3], [4, 5, 6]])
+
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
+def test_daily_standard_deviation(data, expected_standard_deviation):
+    from inflammation.models import s_dev
```
Comment (0liverCrosby) [inline tests/test_models.py:48]: standard_deviation

File: inflammation-analysis.py [hunk 3]
```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
```
Comment (0liverCrosby) [inline inflammation-analysis.py:27]: Test for loop

**Review by 0liverCrosby — APPROVED** [review]
s_dev and for loop test

File: inflammation/models.py (hunk 1 above)
Comment (mjjq) [reply inflammation/models.py]: Updated in 4473808558d74d39b9de03d59ad33628a276384c

**Comment by mjjq** [comment]
Added further test in 7fc0602aba1a9919b321fd0b549a859b6f556add

**Pull request merged**